django_select2/util.py

```diff
@@ -101,7 +106,7 @@ def register_field(key, field):
         if GENERATE_RANDOM_ID:
             id_ = u"%d:%s" % (len(__id_store), unicode(datetime.datetime.now()))
         else:
-            id_ = unicode(hashlib.sha1("%s:%s" % (key, SECRET_SALT)).hexdigest())
+            id_ = hashlib.sha1(six.text_type("%s:%s" % (key, SECRET_SALT)).encode()).hexdigest()
```

Comment:
@codingjoe You are correct it is not necessary to have two different lines for this. I did a quick check in Python 2.7 and Python 3.4. 

```python
patrick@serenity:django-select2 (local)$ ipython
Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
Type "copyright", "credits" or "license" for more information.

IPython 1.2.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import hashlib, six

In [2]: s =six.text_type("%s:%s" % ('key', 'secret salt')).encode()

In [3]: hashlib.sha1(s).hexdigest()
Out[3]: 'a4fb0f227e2b7fcfdf5c50a6ea42d9998144eec3'
```

```python
Python 3.4.0 (default, Apr 11 2014, 13:05:11) 
Type "copyright", "credits" or "license" for more information.

IPython 2.3.1 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import hashlib, six

In [2]: s =six.text_type("%s:%s" % ('key', 'secret salt')).encode()

In [3]: hashlib.sha1(s).hexdigest()
Out[3]: 'a4fb0f227e2b7fcfdf5c50a6ea42d9998144eec3'
```